### DISCUSSION TITLE
How to find and replace to refactor a variable in a project?

### DISCUSSION THREAD

**Discussion by devstefancho** [LunarVim/LunarVim#3627] [Q&A] [answered]
I've currently try to switch back to neovim(lunarvim) from webstorm with ideavim

Currently In my project I often rename varibles
so I want to know how to rename a varible at once in all project files

Please share your knowledge to me
Thanks!
(I'm familiar with using vim but little knowing about plugins)

**Comment by LostInTheLogs** [comment, marked as answer]
For renaming variables use `leader l r`

**Reply by brneor** [reply]
Just to add, not all LSP support rename. For this scenario, I'm currently using [nvim-spectre](https://github.com/nvim-pack/nvim-spectre).

**Reply by stevenroose** [reply]
Where are we supposed to find out about these?

**Reply by emzxcv** [reply]
But i find that it only replaces it in the current file and not for files across the project. How do i rename across multiple files in a project with lunarvim shortcuts?
